static-assets/components/cstudio-common/common-api.js

```diff
@@ -3079,10 +3079,10 @@ var nodeOpen = false,
             getPermissionsServiceUrl: "/api/1/services/api/1/security/get-user-permissions.json",
             lookupAuthoringRoleServiceUrl : "/api/1/services/api/1/security/get-user-roles.json",
             verifyAuthTicketUrl: "/api/1/services/api/1/user/validate-token.json",
-            getUserInfoServiceURL: "/api/2/user",
+            getUserInfoServiceURL: "/api/2/users",
             validateSessionUrl: "/api/1/services/api/1/security/validate-session.json",
             logoutUrl: "/api/1/services/api/1/security/logout.json",
-            getLogoutInfoURL: "/api/2/user/logout/sso/url",
+            getLogoutInfoURL: "/api/2/users/logout/sso/url",
```

Comment:
This is a bug in the impl. It should be `/api/2/users/me/logout/sso/url`. If this is a bug in BE, file a ticket for Jose.

Reply:
Done